packages/commonwealth/client/scripts/views/components/proposals/vote_listing.tsx

```diff
@@ -74,6 +74,17 @@ export const VoteListing = (props: VoteListingProps) => {
     );
   }
 
+  const VoterInfo = ({ voter, shouldShowPopover = true }) => {
+    return (
+      <User
+        userAddress={voter.address}
+        userChainId={voter.chain?.id || voter.profile?.chain}
+        shouldLinkProfile
+        shouldShowPopover={shouldShowPopover}
+      />
+    );
+  };
```

Comment:
The abstraction was only for some cleanup.

> Or instead of creating new component you can do some func like

Yes we could, buts in code its used sequentially like this (notice the shouldShowPopover prop)

```
<VoterInfo voter={vote.account} />
<VoterInfo voter={vote.account} shouldShowPopover={false} />
<VoterInfo voter={vote.account} shouldShowPopover={false} />
<VoterInfo voter={vote.account} />
```